semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -2325,6 +2448,267 @@ language constructs and idioms. However, but we do not yet have a specification
 that comprehensively describes how Scala language features map onto symbol
 occurrences. We intend to improve on this in the future.
 
+<a name="scala-synthetic"></a>
+
+#### Synthetic
+
+The following is an exhaustive list of the kinds of synthetics that are generated by the
+Scala compiler plugin, along with examples.
+
+Synthetics generated from represent for loops are quite massive,
+so we use a shorthand notation for them.
+In that notation, OriginalTree trees are represented by `orig(<code>)` and other trees are represented
+by their syntax.
+
+<table>
+<tr>
+  <td width="220px"><b>Category</b></td>
+  <td><b>Source</b></td>
+  <td><b>Synthetic</b></td>
+</tr>
+<tr>
+  <td valign="top" rowspan="2">
+
+Inferred method calls
+
+  </td>
+  <td>
+      
+```scala
+List(1)
+```
+
+  </td>
+  <td>
+      
+```scala
```

Comment:
Wow, I didn't know you could embed markdown code listings inside HTML markup inside markdown files 😍 I don't want to write `&gt;` ever again 😂